library/templates/dcp_mappluto.yml

```diff
@@ -51,15 +40,15 @@ dataset:
       - dcp_colp
       - lpc_historic_districts
       - lpc_landmarks
-      - dcp_cdboundaries
+      - dcp_cdboundaries_wi
```

Comment:
Why are you switching the dcp_councildistricts to include the _wi? Are there not other columns that have _wi?